### DISCUSSION TITLE
How to serve compressed dll for wasm app?

### DISCUSSION THREAD

**Discussion by deepbluezen** [vercel/vercel#5091] [Help]
Issue: .dll files are served without compression which is unexpected behaviour given support page (https://vercel.com/docs/edge-network/compression). Note the mime type is also application/x-msdownload rather than expected application/octet-stream

wasm files seemed to be compressed and served as expected but Blazor apps etc. can need to reference dll as well.

Is there any settings I can customise to ensure dll are served appropriately? I couldn't see anything relevant in support section.

Thanks in advance for any guidance.

**Comment by mununki** [comment]
Same issue here.
My game web app is served with .wasm and .pck file. The .wasm seems to be compressed and served, but not .pck file. Is there any configuration for setting .pck file to be compressed?
